File: classes/parte-9-ingenieria-de-software-poliglota/143-dependencias-versiones-y-lockfiles/implementaciones/java/Main.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] v = br.readLine().trim().split("\\.");
        System.out.println("mayor=" + Integer.parseInt(v[0]) + " menor=" + Integer.parseInt(v[1]) + " parche=" + Integer.parseInt(v[2]));
    }
}

File: classes/parte-9-ingenieria-de-software-poliglota/143-dependencias-versiones-y-lockfiles/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	v := strings.Split(strings.TrimSpace(line), ".")
	ma, _ := strconv.Atoi(v[0])
	me, _ := strconv.Atoi(v[1])
	pa, _ := strconv.Atoi(v[2])
	fmt.Printf("mayor=%d menor=%d parche=%d\n", ma, me, pa)
}
